src/CoreFoundation/CFProxySupport.cs

```diff
@@ -890,13 +901,13 @@ static Uri GetProxyUri (CFProxy proxy, out NetworkCredential credentials)
 					return null;
 				}
 				
-				string username = proxy.Username;
-				string password = proxy.Password;
-				string hostname = proxy.HostName;
-				int port = proxy.Port;
+				var username = proxy?.Username;
+				var password = proxy?.Password;
+				var hostname = proxy?.HostName;
+				var port = proxy?.Port;
```

Comment:
I don't _love_ the usage of var here where the right hand size doesn't make it obvious if Port is a int or a string for example, but I think it's _fine_ enough.